src/governance.h

```diff
@@ -25,32 +25,29 @@
 #include <stdio.h>
 #include <string.h>
 
-using namespace std;
+class CGovernanceManager;
+class CGovernanceObject;
+class CGovernanceVote;
 
 static const int GOVERNANCE_OBJECT_UNKNOWN = 0;
 static const int GOVERNANCE_OBJECT_PROPOSAL = 1;
 static const int GOVERNANCE_OBJECT_TRIGGER = 2;
 
-class CGovernanceManager;
-class CGovernanceObject;
-class CGovernanceVote;
-class CNode;
-
 static const CAmount GOVERNANCE_PROPOSAL_FEE_TX = (0.33*COIN);
 
 static const int64_t GOVERNANCE_FEE_CONFIRMATIONS = 6;
 static const int64_t GOVERNANCE_UPDATE_MIN = 60*60;
 
-extern std::map<uint256, int64_t> mapAskedForGovernanceObject;
-extern CGovernanceManager governance;
-
 // FOR SEEN MAP ARRAYS - GOVERNANCE OBJECTS AND VOTES
 static const int SEEN_OBJECT_IS_VALID = 0;
 static const int SEEN_OBJECT_ERROR_INVALID = 1;
 static const int SEEN_OBJECT_ERROR_IMMATURE = 2;
 static const int SEEN_OBJECT_EXECUTED = 3; //used for triggers
 static const int SEEN_OBJECT_UNKNOWN = 4; // the default
 
+extern std::map<uint256, int64_t> mapAskedForGovernanceObject;
+extern CGovernanceManager governance;
+extern CCriticalSection cs_budget;
```

Comment:
Where did this come from ?  I had gotten rid of cs_budget and it's not present in the current 12.1 HEAD sources anywhere.